package.json

```diff
@@ -14,6 +14,7 @@
     "express-session": "^1.15.6",
     "morgan": "^1.9.1",
     "nodemailer": "^6.2.1",
+    "nodemon": "^1.19.4",
```

Comment:
Let's not add trivial extensions to our team branches, only use them on your local machine if you feel like they are helpful.